guides/common/assembly_configuring-keycloak-external-authentication-with-cac-cards.adoc

```diff
@@ -0,0 +1,18 @@
+[id="configuring-keycloak-authentication-with-cac-cards_{context}"]
```

Comment:
Lets remove this file for now. CAC card configurations are not yet documented. 

Let me put some context here:

For CAC card:

End User browser configuration:
1) Install required packages
$ yum install opensc -y
2) Install Firefox if not installed.
On Linux system (Fedora), launch Firefox and go to preferences. In Preferences tab go to option ‘Privacy and Security’. On this pane, open ‘Security Devices’. 
3) From ‘Security Devices’ click on ‘Load’ and select ‘/usr/lib64/pkcs11/opensc-pkcs11.so’. 
4) If the Smart Card is connected to system, restart ‘pcscd’ service.
5) Try to access Satellite. It should prompt to enter the PIN of the Smart Card and then should give an option to choose the certificate for authentication.
5) Browser will present this certificate to Keycloak. Once authenticated, browser will redirect back to Satellite and the user would be logged in.

RHSSO/Keycloak configuration:
1) Go to the ‘Authentication’ page and in ‘Flows’ tab, select ‘Browser’. Copy this flow by clicking ‘Copy’ command. Name it ‘Copy of Browser’.
2) In the newly created flow(Copy of Browser) delete ‘Username Password Form’ and ‘OTP Form’.
3) Click on ‘Add execution’ and select provider ‘X509/Validate Username Form’. Save this execution flow. Now select ‘ALTERNATIVE’ radio button for newly added execution flow.
4) To configure ‘X509/Validate Username Form’ click on ‘Actions’ and select config. Give ‘Alias’ as per wish. Select ‘User Identity Source’ as ‘Subject’s Common Name’, select ‘User mapping method’ as ‘Username or Email’.
5) Now go to ‘Bindings’ tab and select ‘Copy of Browser’ in ‘Browser Flow’.

Reply:
Thanks for the steps @rabajaj0509. I have implemented them  in `guides/common/modules/proc_configuring-keycloak-authentication-with-cac-cards.adoc`. Please review. Is the order of the procedure correct? Thank you